### ISSUE TITLE
Add a when to use the latent model vignette

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epidist#486]
As #426 has switched all our public facing examples over to the maringal model we may want a new vignette which deals with when a user might want to use the latent model (likely small data with a lot of custom settings). This would be a helpful section for the `primarycensored` paper so contributing this would naturally lead to authorship there.

**Comment by athowes** [comment]
Good idea. Are there data which have a lot of information we might be able to use for this?

**Comment by seabbs** [comment]
Added some docs to the individual models to help support this choice. Fully closing this requires a larger scale evaluationf

**Comment by athowes** [comment]
Do you imagine this being:

1. A simulation study comparison of the two model types
2. Finding >1 situation where the latent model is preferable and highlighting it

In my opinion for the vignette here it should be 2. and for the `primarycensored` paper it should be 1.

Perhaps the situation is that we don't exactly know which situation is needed for 2, and so we could do with already having 1 available so we could just cherry pick from it.

I think another possibility could be that there is just some kind of situation where it actually isn't really possible to use the marginal model. In that case, all that it would require for the latent model to be preferable is that it works.

**Comment by seabbs** [comment]
This can draw on, ref and summarise some of the results from the primarycensored paper
